db/migrations/000004_new_tables.up.sql

```diff
@@ -5,43 +5,8 @@ CREATE TABLE cms_user (
   insert_time TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
   last_update_time TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
 );
--- TODO: Is tit worth using a more specific index, e.g. on (Issuer,Subject)?
+-- TODO: Is it worth using a more specific index, e.g. on (Issuer,Subject)?
 CREATE INDEX idx_user_body ON cms_user USING GIN (body);
```

Comment:
I'm not sure we need a GIN index on the entire body.  If we need them on specific attributes, we can create them on those attributes: "using gin ((body->'attrName'))". But at least so far, I can't think of where we reference body attributes in WHERE clauses.

Reply:
I'm fine either way. This shouldn't be a big table so it probably doesn't matter much.